### ISSUE TITLE
Has anybody downloaded the safe/suggestive images at least maybe 2 years ago?

### ISSUE THREAD

**Issue by Twi-Hard** [woodenphone/Derpibooru-dl#35]
The only people I could think of that could have the image I am looking for are people that have used this tool. There is an image from a comic that was removed from Derpibooru for seemingly no reason.. I spoke to the artist he didn't keep a copy of his work after he moved on and it wouldn't have broken any rules.. :/
It's also nowhere else on the internet.. Anyways, the image would be between 590000 and 593500.
I'm sorry if I shouldn't be asking this but it has been stressing me out so i thought I'd ask.

It could have been slightly less that 2 years ago but I'm not sure.

**Comment by liamwhite** [comment]
This is not an issue with the project, closing.

That said, I may be able to help you find it. Provide any details you can about this comic and I will attempt to look for it.

**Comment by Twi-Hard** [comment]
It's from [Caught in the Storm](https://derpibooru.org/tags/caught+in+the+storm) by [garretthegarret](https://derpibooru.org/tags/artist-colon-garretthegarret). I've used advanced searches on Google, I've looked through archive.org and archive.is on the artists DeviantArt. I also used a tool to search about 15 different image sites similar to Derpibooru. I've also asked the artist and a fanfic writer. Asking here seemed like my last option.
